py/ringbuf.h

```diff
@@ -95,5 +95,7 @@ static inline size_t ringbuf_avail(ringbuf_t *r) {
 int ringbuf_get16(ringbuf_t *r);
 int ringbuf_peek16(ringbuf_t *r);
 int ringbuf_put16(ringbuf_t *r, uint16_t v);
+int ringbuf_read(ringbuf_t *r, void *data, size_t data_len);
```

Comment:
There should be a blank line before this new function, because the comment above about endianness doesn't apply to these.